changelog.md

```diff
@@ -26,3 +26,8 @@
 
 ## 0.5.2
 - Fix `RegisterModule(Mock<T> mock)` by removing `new()` restriction
+
+## Unreleased
```

Comment:
- Using `Messerli.CompositionRoot`
- Converted `MockModuleBuilder`s mock functionality to `ModuleBuilderMockExtension`
- Renamed `MockContainerBuilder` to `ContainerBuilderMockExtension`.
- Delete `MockModule` and `MockModuleBuilder`

Reply:
Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/20/commits/017bbea67da2efd3d482fb4328d281249606b7f2
Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/20/commits/dd4e600708c40895dc3311ea4071e4861f104b75